.changeset/fuzzy-seahorses-brake.md

```diff
@@ -0,0 +1,25 @@
+---
+"@comet/cms-admin": major
+---
+
+Update Scope Picker to display all scope dimensions in a single dropdown
```

Comment:
I believe the title line needs to be separated from the body with an empty line, just like in a commit message. 

```suggestion
Update Scope Picker to display all scope dimensions in a single dropdown

```